### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 27

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#203] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152254317-a47c137d-f2ce-4d59-b08f-35d3c4d1f287.png)

**Comment by Exyss** [comment]
- 14 lettere totali
- 2 lettere M
- 2 lettere A
- 2 lettere O
- 2 lettere R

**Risposte:**
1. 14! / (2! * 2! * 2! * 2!)
2. (12 * 13!) / (2! * 2! * 2! * 2!)
3.  #(Iniziano_AM U Finiscono_AM) = ((2 * 2 * 12!) / (2! * 2! * 2! * 2!)) + ((12! * 2 * 2) / (2! * 2! * 2! * 2!)) - ((2 * 2 * 10! * 1 * 1) / (2! * 2! * 2! * 2!))

**Comment by matypist** [comment]
_Problema simile (anche per approcci risolutivi) all'esercizio 26 (#202 )_
